Sources/Exporters/DatadogExporter/ExporterConfiguration.swift

```diff
@@ -37,15 +38,15 @@ public struct ExporterConfiguration {
     /// Option to add a custom prefix to all the metrics sent by the exporter
     var metricsNamePrefix: String?
 
-    public init(serviceName: String, resource: String, applicationName: String, applicationVersion: String, environment: String, clientToken: String?, apiKey: String?, endpoint: Endpoint, uploadCondition: @escaping () -> Bool, performancePreset: PerformancePreset = .default, exportUnsampledSpans: Bool = true, exportUnsampledLogs: Bool = true, hostName: String? = nil, metricsNamePrefix: String? = "otel") {
+    public init(serviceName: String, resource: String, applicationName: String, applicationVersion: String, environment: String, apiKey: String, endpoint: Endpoint, source: String = "otel-swift-exporter", uploadCondition: @escaping () -> Bool, performancePreset: PerformancePreset = .default, exportUnsampledSpans: Bool = true, exportUnsampledLogs: Bool = true, hostName: String? = nil, metricsNamePrefix: String? = "otel") {
```

Comment:
Wasn't `source` set to `"ios"` before? 
I'm not exactly sure about the impact of using `"otel-swift-exporter"` 💭 - I think this won't install default "iOS" processing pipelines (in Logs) when the first data comes in.

Reply:
Oh, I was not aware of that. I will change it back